layers/+window-management/exwm/config.el

```diff
@@ -0,0 +1,11 @@
+(defvar exwm--terminal-command "xterm"
+  "Terminal command to run.")
+
+(defvar exwm--locking-command "lock"
```

Comment:
I don't think `lock` makes sense as a default, but I don't know what would be a good default value.